workflows/scripts/insilico_coverage/insilico_overall_coverage.py

```diff
@@ -0,0 +1,66 @@
+#!/apps/bio/software/anaconda2/envs/mathias_general/bin/python3.6
+import os
+import csv
+import sys
+import argparse
+import statistics
+
+def count_bases_in_threshold(coverage_list, threshold):
+    total_positions = len(coverage_list)
+    count_positions = 0
+    for position in coverage_list:
+        if position >= threshold:
+            count_positions += 1
+    percent_covered = float(count_positions*100) / total_positions
+    percent_covered = float("{0:.3f}".format(percent_covered))
+    return percent_covered
+
+def calculate_mean(coverage_list) -> float:
+    mean_calc = sum(coverage_list) / len(coverage_list) 
+    mean = float("{0:.3f}".format(mean_calc))
+    return mean
+
+def calculate_median():
+    pass
```

Comment:
is this used for anything?

Reply:
No, I this script was copied from WOPR. I will remove.